zeroconf/test.py

```diff
@@ -374,7 +381,7 @@ def on_service_state_change(zeroconf, service_type, state_change, name):
 
         # wait until the browse request packet has maxed out in size
         sleep_count = 0
-        while sleep_count < 100 and longest_packet_len < r._MAX_MSG_ABSOLUTE - 100:
+        while sleep_count < 100 and longest_packet_len < (5 * r._MAX_MSG_TYPICAL) - 100:
```

Comment:
Does 5 have some specific meaning in this context?

Reply:
No, I changed it back since it's really immaterial now, and I explain why it doesn't matter in a comment.